addons/account/views/account_account_views.xml

```diff
@@ -47,8 +47,7 @@
                                         <div>
                                             <field name="name"
                                                    placeholder="e.g. Current Assets"
-                                                   class="oe_inline"
-                                                   style="width:80% !important;"/>
+                                                   style="width:130%"/>
```

Comment:
We try to avoid using "style" on field because inline css ( html style attribute ) overrides css rules in style tag and css file. Instead of that you can either try using boostrap in the class, or if you don't achieve with boostrap you can make a custom class.

Also while testing the fix, it seems that when having a large screen it's not really great :/ the spec need to be rethink a bit
![image](https://github.com/odoo/odoo/assets/109144554/550482e6-46ca-4758-9e99-edd9b5621d04)